### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by jwiggins** [celiagg/celiagg#4]
Windows probably works, but it hasn't been tested at all

**Comment by jdeschenes** [comment]
Before closing that task I think there are a few other lingering issues that should be resolved:

- [ ] appveyor builds should be integrated
- [ ] clean should be tweaked

**Comment by jwiggins** [comment]
@jdeschenes: I think this can be closed now but I don't know what you mean by "clean should be tweaked".

**Comment by jdeschenes** [comment]
Hi @jwiggins, it's been a while! This is an embarassingly terse comment, sorry about that. I am afraid I do not fully recall.

My guess is it could be related to the clean method

```bash
python setup.py clean
```

It does not fully clean the build artifacts(e.g.` _celiagg.cpp` is not cleaned up)
This was a pain point for me a few years back. Sometimes, even though I made some code changes, no change were detected and thus no cythonization occurred. I was compiling against an old version of the code. I was manually deleting the cython generated files. Cleaning is significantly more convenient.

I had implemented some logic to delete the cython generated files when invoking `clean` so that I could be sure that I was using the latest and greatest on some of my projects.

The pandas project is doing something that resembles what I implemented in my projects.

As far as I can tell, this is not a specific problem with Windows.

I hope that was helpful.
